consensus/abci.go

```diff
@@ -226,7 +240,8 @@ func (abci *TendermintABCI) DeliverTx(txBytes []byte) tmtAbciTypes.ResponseDeliv
 		return tmtAbciTypes.ResponseDeliverTx{Code: 1, Log: "INVALID_TX"}
 	}
 
-	abci.logger.Info("Delivering TX", "hash", tx.Hash().String(), "nonce", tx.Nonce(), "cost", tx.Cost(), "gas", tx.Gas(), "gas_price", tx.GasPrice())
+	abci.logger.Info("Delivering TX", "hash", tx.Hash().String(), "nonce", tx.Nonce(), "cost", tx.Cost(), 
```

Comment:
[Comment] Adding blockstate `height` for better monitoring of the action made

Reply:
[Comment] Nice. Just please keep it in the same line next time like it was. Or add EVERY argument on a new line (like u did in checkTx) but not a combination of it. It's impossible to read :) (I am adding it back to one line, no worries)